eventconsumer/src/main/scala/com/gu/notifications/events/SqsLambda.scala

```diff
@@ -56,4 +67,13 @@ class SqsLambda extends RequestStreamHandler {
     }
   }
 
+  private def processSqsEvent(event: SqsEvent) = event.Records.foldLeft(Future.successful[SqsAttemptCount](SqsAttemptCount(NotificationCount(0, 0), 0, 0))) {
```

Comment:
I'm having a hard time to understand what this function is doing.
 - If it's a function with side effect, make it clear by adding the return type `Unit` otherwise a return type would help me a lot here
 - Because the definition of `SqsAttemptCount`, it's hard to understand what the counts are. Either define `SqsAttemptCount.empty` or use named parameters
 - I'm wondering if there wouldn't be a way to split the processing from the counting as it seems like a lot is happening in the same place